scripts/translation/lib/QaFileInfo.php

```diff
@@ -54,7 +54,7 @@ public static function cacheSave( array $itens )
         foreach( $itens as $item )
         {
             $line = array( $item->sourceHash , $item->targetHash , $item->sourceDir , $item->targetDir , $item->file , $item->days );
-            fputcsv( $fp , $line );
+            fputcsv( $fp , $line, ",", "\"", "\\" );
```

Comment:
I'd rather we not use the proprietary escape mechanism:
```suggestion
            fputcsv( $fp , $line, ",", "\"", "" );
```
Also I'm pretty sure we use stuff that require PHP 8 (PhD does at least)

Reply:
phd requires PHP 8.1 and the website requires PHP 8.2 so I think it's pretty safe to say that docbase doesn't support PHP <= 8.0 anymore.

Reply:
This serialization is something I'm about to change in  "Extricate qaxml tools from lib revcheck"/"Test and modify qaxml tools to be used in more languages" itens of issue 212, and the simple `fputcsv( $fp , $line, escape: "" );`, as now, is ok.